integrationtests/src/test/java/org/neo4j/procedure/UserFunctionIT.java

```diff
@@ -712,14 +713,26 @@ public void shouldListAllFunctions() throws Throwable
         "| 'org.neo4j.procedure.simpleArgument'                | 'org.neo4j.procedure.simpleArgument(someValue :: INTEGER?) :: (INTEGER?)'                                                                                    | ''                      |" + lineSeparator() +
         "| 'org.neo4j.procedure.squareDouble'                  | 'org.neo4j.procedure.squareDouble(someValue :: FLOAT?) :: (FLOAT?)'                                                                                          | ''                      |" + lineSeparator() +
         "| 'org.neo4j.procedure.squareLong'                    | 'org.neo4j.procedure.squareLong(someValue :: INTEGER?) :: (INTEGER?)'                                                                                        | ''                      |" + lineSeparator() +
+        "| 'org.neo4j.procedure.sum'                           | 'org.neo4j.procedure.sum(numbers :: LIST? OF NUMBER?) :: (NUMBER?)'                                                                                          | ''                      |" + lineSeparator() +
```

Comment:
Actually it's intentional, as users will have namespace functions where the simple function name clashes with built in ones. Which is fine as we have the namespace.

Actually that was the reason for this fix, the clash. And we will not force users to make up awkward names for common operations.